### PR TITLE
Display of roles required for a service in Swagger-UI

### LINKED ISSUE #616: Display of roles required for a service in Swagger-UI
As we have now introduced RBAC, it will be informative to also display the 'allowed roles' for each method in the Swagger UI.

Some options - https://stackoverflow.com/questions/76153735/how-to-display-required-user-roles-in-for-openapi-ui

### PR DESCRIPTION
**Pull request by baixinsui**
fixed #616

### CODE REVIEW

File: modules/security/src/main/java/org/eclipse/xpanse/modules/security/zitadel/config/UserRoleDescriptionCustomizer.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import org.springframework.security.access.annotation.Secured;
+import org.springframework.stereotype.Component;
+import org.springframework.web.method.HandlerMethod;
+
+/**
+ * Custom Operation of Swagger-UI to show required roles.
+ */
+@Profile("zitadel")
+@Component
+public class UserRoleDescriptionCustomizer implements OperationCustomizer {
```
Comment (swaroopar) [inline modules/security/src/main/java/org/eclipse/xpanse/modules/security/zitadel/config/UserRoleDescriptionCustomizer.java:23]: @baixinsui do we really need this new annotation?
Can we not simply add the description when `secured` annotation is available in class or method level?

**Review by swaroopar — CHANGES_REQUESTED** [review]
comment added

File: modules/security/src/main/java/org/eclipse/xpanse/modules/security/zitadel/config/UserRoleDescriptionCustomizer.java (hunk 1 above)
Comment (baixinsui) [reply modules/security/src/main/java/org/eclipse/xpanse/modules/security/zitadel/config/UserRoleDescriptionCustomizer.java]: I removed it and appended the required role to description of Operation.
image

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**